Makefile

```diff
@@ -281,11 +281,12 @@ install-contour-release: | setup-kind-cluster ## Install the release version of
 	./test/scripts/install-contour-release.sh $(CONTOUR_UPGRADE_FROM_VERSION)
 
 .PHONY: e2e
-e2e: | setup-kind-cluster run-e2e cleanup-kind ## Run E2E tests against a real k8s cluster
+e2e: | setup-kind-cluster load-contour-image-kind run-e2e cleanup-kind ## Run E2E tests against a real k8s cluster
```

Comment:
This breaks running the tests locally. Could this maybe check for the condition and default to not running the incluster? Or maybe we can buy into the in-cluster only when running in GH actions?

Reply:
ah yeah good call, I'll work on that so you can still just run `make e2e` and things Just Work ™️